docs/src/references/bridged-topics.md

```diff
@@ -64,6 +64,17 @@ The Azure topics are prefixed by `az/`.
  Any message published by Azure on one the subtopics of `devices/{device_id}/messages/devicebound/#`
  is republished here.
  
+ ## AWS MQTT Topics
+MQTT clients on Thin Edge device must use the below topics to communicate with the AWS cloud.
+The AWS topics are prefixed by `aws/`.
+
+ * `aws/messages/`  - Use this topic to send the messages from device to cloud.
+ The messages are forwarded to the AWS topic named `thinedge/devices/{device_id}/messages/`
+ where {device_id} is the Thin Edge device id.
+
+ * `aws/messages/devicebound/#` - Use this topic to subscribe for the messages that were sent from cloud to device.
+ Any message published by AWS on one the subtopics of `thinedge/devices/{device_id}/messages/devicebound/#`
```

Comment:
This naming scheme needs to be revised. See my comment on the bridge configuration.